website/docs/guides/best-practices/environment-setup/1-env-guide-overview.md

```diff
@@ -0,0 +1,67 @@
+- Illustrate these recommendations with comprehensive examples
+- At each stage, explain *why* we recommend the approach that we do, so that you're equipped to decide when and where to deviate from these recommendations to better fit your organization’s unique needs
+
+:::info
+☁️ This guide focuses on architecture for **dbt Cloud**. However, similar principles apply for developers using dbt Core. Before diving into this guide we recommend taking a look at our **[dbt Cloud environments](docs/collaborate/environments/dbt-cloud-environments)** page for more context.
+
+:::
+
+### How many environments do I really need?
+
+Environments define the way that dbt will execute your code, including
+
+- the **version of dbt** that will run.
+- the **version of your code** to be executed.
+- the **connection information** for your warehouse.
+- In dbt Cloud, there are **two types of environments:**
+  - **Development** - the environment settings in which you work in the IDE on a development branch.
+  - **Deployment** - the environment settings in which a dbt Cloud job runs.
+
+In this guide, we’re going to focus on **deployment environments**, which determine how your project is executed when a **dbt Cloud job executes**.
+
+Depending on your desired outcome, the number of deployment environments in your dbt Cloud Project will vary. Let’s dive in to see how many deployment environments works best for you.
+
+| Setup option | Works well if you | Relative complexity level  |
+| --- | --- | --- |
+| One deployment environment | - only scheduled runs for one set of data object - development branches are merged directly to main | Low |
+| Many Deployment Environments | - feature branches move through several promotion stages | High |
+
+**One deployment environment** 
+
+We usually recommended folks start with the basics; having one deployment environment is usually the simplest and most maintainable approach to start. This approach works well if:
+
+- You only need to have **scheduled jobs running in a single environment** within your data warehouse.
+- You use a **single primary branch** and follow a direct promotion (**Dev —> Prod**) strategy
+
+With this option, your production jobs and your [Slim CI jobs](docs/deploy/cloud-ci-job) that ensure code integrity are managed within one single deployment environment.
+
+**Many deployment environments** 
+This approach adds a bit more complexity and may slow down the development process, but adds a layer of security that can be worth the tradeoff. This approach works well if:
+
+- Your organization maintains **several long-lived git branches** to control how and when changes are tested and promoted to production.
+  - Some orgs follow a **Dev —> QA —>  Prod release cycle** — if that sounds like your org, this approach is probably right for you.
+- The **output of your dbt project is an input to other systems**, and you need to test and validate any changes in a pre-production environment.
```

Comment:
Can you give an example here where a dbt project is an input to other systems, so the sentence reads, "...is an input other systems, such as _____, and you need to test and validate any changes...". I think that can help contextualize and solidify this bullet.

Reply:
@kbrock91 @graciegoheen i think this was something written in your original draft -- do you have an example here that could help clarify this bullet point?